USA/CA/citrus_college/citrus_college_annual_scraper.py

```diff
@@ -1,61 +1,13 @@
-import requests
 import os
-from bs4 import BeautifulSoup
-import urllib
-import re
-import time
-
+import configs_annual
 import sys
 from pathlib import Path
 
 p = Path(__file__).resolve().parents[3]
 sys.path.insert(1, str(p) + "/common")
-from bs_scrapers.get_files import get_files
-
-__noted__ = "fixes shamelessly stolen from dunnousername without credit"  # Just don't delete this
-webpage = "https://www.citruscollege.edu/campussafety/Pages/AnnualReports.aspx"
-"""
-Click the links that lead to the files, and copy their paths. **NOTE:** Ensure that files all match paths, otherwise remove a level until they match
-Also ensure that domain stays the same
-Verify on page that the href to the file contains the domain, if it doesn't, uncomment domain
-"""
-web_path = "/campussafety/Documents/reports/"
-domain = "https://www.citruscollege.edu/"
-sleep_time = 5  # Set to desired sleep time
+from list_pdf_scrapers.list_pdf_v2 import list_pdf_v2
 
 cur_dir = os.getcwd()
```

Comment:
You can should be able to replace these lines with `save_dir = "./data/annual/"`